pkg/pod/pod.go

```diff
@@ -50,6 +50,7 @@ func DeletePods(c client.Client, logger logr.Logger, pl *corev1.PodList, ignoreA
 			logger.Info(fmt.Sprintf("Applying pod deletion drain strategy to pod %v/%v", p.Namespace, p.Name))
 			err := c.Delete(context.TODO(), &p, options...)
 			if err != nil {
+				logger.Error(err, "failed to delete the pod %v/%v", p.Namespace, p.Name)
```

Comment:
Either this needs to be wrapped in a sprintf or the key/values need to be specified properly.

```suggestion
				logger.Error(err, fmt.Sprintf("failed to delete the pod %v/%v", p.Namespace, p.Name))
```

or

```suggestion
				logger.Error(err, fmt.Sprintf("failed to delete the pod", "namespace", p.Namespace, "name", p.Name))
```

Reply:
Thank you @mrbarge I have applied the suggestion in my last commit.